mindsdb/interfaces/storage/model_fs.py

```diff
@@ -33,6 +33,18 @@ def status_set(self, status, status_info=None):
             rec.data = status_info
         db.session.commit()
 
+    def training_state_set(self, current_state_num, total_states, state_name):
```

Comment:
Maybe is better not having all parameters required, for example I want to do:
`training_state_set(1, 10)`
then
`training_state_set(2)`
or want to use only state names `training_state_set(name="almost done")`
@paxcema, what do you think?

Reply:
I guess this way it could be more flexible. For example maybe a handler would like to give "sub-updates" within a phase, so with this they could simply do `training_state_set(state_name='now doing X')`.

Seems like it's not a huge change (we can partially update the record with the subset of provided values), so I would be in favor just to see how implementers use it.

Reply:
Done